### PR TITLE
Update CaT-service.yaml

### PR DESCRIPTION
**Pull request by ronin1972**
Update to change project confirm endpoint to a change name endpoint and provide the equivalent for an event.

### CODE REVIEW

File: cat/CaT-service.yaml [hunk 1]
```diff
@@ -554,27 +554,25 @@ paths:
...
-  /tenders/projects/{proc-id}/confirm:
+  /tenders/projects/{proc-id}/name:
     parameters:
       - $ref: '#/components/parameters/ProcurementParam'
-    post:
-      deprecated: true
-      description: Confirm the details transfered or created for the agreement are what the user wants.
+    put:
+      deprecated: false
+      description: Replace the name transfered or created for the procurement automatically with what the user wants.
```
Comment (adrianmilne) [inline cat/CaT-service.yaml:562]: Very minor observation - `transfered` should be `transferred`

File: cat/CaT-service.yaml [hunk 2]
```diff
@@ -1359,6 +1358,42 @@ paths:
             - buyer
       summary: Create a Procurement Event
       
+  /tenders/projects/{proc-id}/{proc-id}/events/{event-id}/name:
```
Comment (adrianmilne) [inline cat/CaT-service.yaml:1361]: Looks like double `{proc-id}`?

File: cat/CaT-service.yaml [hunk 3]
```diff
@@ -1302,7 +1300,7 @@ paths:
       description: Get Event's for this Project. In OCDS this is the related cases of Type
       tags:
         - Manage Procurement Project
-      operationId: caseRFXs
+      operationId: RFXs
```
Comment (davidhughes1-ccs) [inline cat/CaT-service.yaml:1303]: Really trivial comment (feel free to ignore). Would this not be 'Get RFxs' as it is the name of the operation not the object. Even more trivial, small 'x'.
Just realised, operation names is one thing not covered by the standards. e.g. camel case, snake case?

**Review by davidhughes1-ccs — COMMENTED** [review]
Generally also agree with Adrian's points.

File: cat/CaT-service.yaml (hunk 1 above)
Comment (ronin1972) [reply cat/CaT-service.yaml]: Corrected and release to the branch

File: cat/CaT-service.yaml (hunk 2 above)
Comment (ronin1972) [reply cat/CaT-service.yaml]: Corrected and release to the branch

File: cat/CaT-service.yaml [hunk 4]
```diff
@@ -554,27 +554,25 @@ paths:
...
+      description: Replace the name transferred or created for the procurement automatically with what the user wants.
       tags:
         - Create Procurement Project
       operationId: ConfirmProcurementProject
       requestBody:
         content:
           application/vnd.api+json:
             schema:
-              $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/Release'
+              properties:
```
Comment (tommyb82) [inline cat/CaT-service.yaml:570]: @ronin1972 Could we add the following to this inline schema def please, to aid the code generation:
```
type: object
title: ProcurementProjectName
```
Without this, we have no wrapper object to use in the generated code and it makes things surprisingly awkward (as @adrianmilne has been discovering!).  Feel free to change the title.  Same required for the event name update 👍

File: cat/CaT-service.yaml [hunk 5]
```diff
@@ -1359,6 +1358,42 @@ paths:
...
+      deprecated: false
+      description: Replace the name created for the event automatically with what the user wants.
+      tags:
+        - Manage Procurement Project
+      operationId: updateRFxName
+      requestBody:
+        content:
+          application/vnd.api+json:
+            schema:
+              properties:
```
Comment (tommyb82) [inline cat/CaT-service.yaml:1375]: Same as above please..

**Review by tommyb82 — CHANGES_REQUESTED** [review]
Hi Nick - couple of tweaks if you don't mind to help us lowly devs out.. 😄

**Review by adrianmilne — APPROVED** [review]
👍  Good stuff - generated objects which are now working on corresponding dev branch

File: cat/CaT-service.yaml (hunk 3 above)
Comment (ronin1972) [reply cat/CaT-service.yaml]: Hi, I think I have made this change as required. I think I am just blocked on you approving this. Should be in last update on this branch. If I have done it right approve and I will then merge.

Thanks

**Review by tommyb82 — APPROVED** [review]
Great!

**Pull request merged**